src/main/scala/org/broadinstitute/dsde/workbench/leonardo/monitor/ClusterDateAccessedActor.scala

```diff
@@ -48,6 +48,7 @@ class ClusterDateAccessedActor(autoFreezeConfig: AutoFreezeConfig, dbRef: DbRefe
   private def updateDateAccessed(clusterName: ClusterName, googleProject: GoogleProject, dateAccessed: Instant) = {
     logger.info(s"Update dateAccessed for $clusterName in project $googleProject to $dateAccessed")
     dbRef.inTransaction { dataAccess =>
+      dataAccess.clusterQuery.clearKernelFoundBusyDateByProjectAndName(googleProject, clusterName)
```

Comment:
I'm not familiar with slick, will this line actually get run? cc @rtitle  (this DBIO is not chained in anything)